addon/styles/addon.scss

```diff
@@ -1,7 +1,204 @@
 @import 'bower_components/bourbon/app/assets/stylesheets/bourbon';
+@import 'node_modules/ember-frost-css-core/scss/frost-core';
+@import 'node_modules/ember-frost-theme/scss/frost-theme';
 
-// Temporarily imported some frost-css code directly here until that is published
-// previously we were importing: node_modules/frost-css/styles/frost/variables;
-@import './frost-css/variables';
+@mixin button-color ($color) {
+  color: $color;
+  svg * {
+    stroke: $color;
+    fill: $color;
+  }
+}
```

Comment:
This little bit of skulduggery is to set the color of icons within buttons to match the styled text color.